src/main/java/seedu/sellsavvy/model/order/OrderList.java

```diff
@@ -144,4 +144,15 @@ private int indexOf(Order target) {
 
         return -1;
     }
+
+    /**
+     * Creates a copy of this {@code OrderList} with new Orders.
+     */
+    public OrderList createCopy() {
+        OrderList orderList = new OrderList();
```

Comment:
Rename orderList to orderListCopy for consistency?

Reply:
sure